nifi-nar-bundles/nifi-aws-bundle/nifi-aws-processors/src/test/java/org/apache/nifi/processors/aws/ml/polly/MockAwsCredentialsProvider.java

```diff
@@ -0,0 +1,24 @@
+package org.apache.nifi.processors.aws.ml.polly;
+
+import com.amazonaws.auth.AWSCredentialsProvider;
+import org.apache.nifi.controller.AbstractControllerService;
+import org.apache.nifi.processor.exception.ProcessException;
+import org.apache.nifi.processors.aws.credentials.provider.service.AWSCredentialsProviderService;
+
+public class MockAwsCredentialsProvider extends AbstractControllerService implements AWSCredentialsProviderService {
```

Comment:
This could be removed and replaced with a Mockito-managed mock instance of `AWSCredentialsProviderService`.